src/search_algo.hpp

```diff
@@ -1251,18 +1209,11 @@ sortMatches(TLocalHolder & lH)
     if ((lH.options.filterPutativeAbundant) &&
         (lH.matches.size() > lH.options.maxMatches))
         // more expensive sort to get likely targets to front
-        myHyperSortSingleIndex(lH.matches, lH.options.doubleIndexing, lH.gH);
+        myHyperSortSingleIndex(lH.matches, lH.gH);
     else
         std::sort(lH.matches.begin(), lH.matches.end());
 
-    double finish = sysTime() - start;
-
-    if (lH.options.doubleIndexing)
-    {
-        appendToStatus(lH.statusStr, lH.options, 1, " done. ");
-        appendToStatus(lH.statusStr, lH.options, 2, finish, "s. ");
-        myPrint(lH.options, 1, lH.statusStr);
-    }
+    // double finish = sysTime() - start;
```

Comment:
we did you comment out the time measuring

Reply:
I think we don't need it anymore but can you double check?

Reply:
it can go away